client/src/components/Sub_ProductDetail/ProductInfo.jsx

```diff
@@ -1,11 +1,25 @@
 /* eslint-disable react/no-array-index-key */
 import React from 'react';
 import SocialMediaShare from './Sub_ProductInfo/SocialMediaShare.jsx';
+import generateStars from '../Sub_RatingsReviews/generateStars.js';
+
+function ProductInfo({ product, reviewMetadata }) {
+  const { ratings } = reviewMetadata;
+  const totalStars = Object.entries(ratings).reduce((
+    (a, b) => a + Number(b[0]) * Number(b[1])), 0);
```

Comment:
I know I did it the same way, but you skipping over the `.map` and going straight into the `.reduce` makes it a little harder to read for me.

Reply:
yes, I initially use.map and .reduce and then i realized i can just use .redue only.  I agree that it's a little harder to understand...